xm-commons-config/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     testImplementation 'junit:junit'
     testImplementation 'org.assertj:assertj-core'
     testImplementation 'org.mockito:mockito-core'
+    testImplementation "org.springframework:spring-test:${versions.springTest}"
```

Comment:
can this version be managed by io.spring.dependency-management ?

Reply:
done